SPECS/pyproject-rpm-macros/macros.aaa-pyproject-srpm

```diff
@@ -0,0 +1,7 @@
+# This file is called macros.aaa-pyproject-srpm
+# to sort alphabetically before macros.pyproject.
+# When this file is installed but macros.pyproject is not
+# this macro will cause the package with the real macro to be installed.
+# When macros.pyproject is installed, it overrides this macro.
+# Note: This needs to maintain the same set of options as the real macro.
+%pyproject_buildrequires(rRxtNwe:C:) echo 'pyproject-rpm-macros' && exit 0
```

Comment:
I don't think the idea behind this file is Fedora-specific. It might be better to just drop this file to avoid confusion. Especially since having this fille installed but not `macros.pyproject` will still not install `pyproject-rpm-macros`.

Reply:
I've removed this